number_of_models: 2

camera:
  number_of_measurements: [15, 20, 0]
  offset: [
    -0.539556, -0.017505, 
    -0.584873, -0.0353828, 
  ]
  covariance: [
    0.000294585, 0.00131743, 0.0,
    0.000267119, 0.00523626, 0.0
  ]

lidar:
  number_of_measurements: [15, 20, 0]
  offset: [
    -0.179899, -0.00320078, 0.0,
    -0.195142, -0.00670272, 0.0
  ]
  covariance: [
    3.12859e-06, 1.1027e-05, 0.0,
    3.1092e-06, 4.65465e-05, 0.0
  ]